demo/webclient/Dockerfile

```diff
@@ -0,0 +1,26 @@
+# Stage 0: Build the frontend using Node.js
+FROM node:18 AS build-stage
+
+# Set the working directory
+WORKDIR /app
+
+# Copy package files and install dependencies
+COPY ./package*.json /app/
```

Comment:
Creating and copying a package-lock.json file could be a good option for managing versioning

Reply:
I don't know we would want to manage versions here , I think that since it is a code made by mediasoup we should keep the version that comes from its repository.  What do you think? @genaris ?

Reply:
I think @lotharking talks about copying `package-lock.json` so the Docker image is frozen to the dependencies we use during development. The script in Dockerfile actually does so (COPY ./package*.json /app/), but there is no uploaded package-lock.json in this PR, so it will always create it when building the image.

I prefer to have control on the dependencies we are using, and upgrade dependencies only if we need so. Therefore, IMHO it will be better to upload your current `package-lock.json` so we are sure that the image will be built with the exact versions you tested today.

Note that, if you want to do so, you will need to remove `.npmrc` file, which is preventing us from using package-lock.

Reply:
Resolved your requirements add package-lock.json and remove .npmrc